components/builder-api-client/src/lib.rs

```diff
@@ -398,7 +398,7 @@ impl Client {
         };
         match result {
             Ok(response) => {
-                if response.status == StatusCode::Ok {
+                if response.status == StatusCode::Created {
```

Comment:
Would be better to check for either Ok or Created in order to be backward compatible (on-prem instances running pre-Actix Builder)

Reply:
Ah I was confused by what you meant by that. I'll get that added.